src/interactive/documentation.ts

```diff
@@ -80,7 +104,9 @@ async function getDocumentation(): Promise<string> {
 
     return await withLanguageClient(
         async languageClient => {
-            return languageClient.sendRequest('julia/getDocAt', getVersionedParamsAtPosition(editor, editor.selection.start))
+            const docAsMD = await languageClient.sendRequest<string>('julia/getDocAt', getVersionedParamsAtPosition(editor, editor.selection.start))
+            const docAsHTML = md.render(docAsMD)
+            return docAsHTML
```

Comment:
I would like to do this HTML conversion in `createWebviewHTML`, but we can tweak this later.